blog/content/post/2021-12-13-secure-jtag-on-imx-soc.md

```diff
@@ -0,0 +1,280 @@
+---
+title: 'Increasing the security of iMX platforms - JTAG fusing'
+abstract: 'JTAG helps a lot of engineers during product development. It also may
+          be helpful for hackers. I tell you why and how to increase JTAG
+          security in your product'
+cover: /img/secure_jtag_cover.png
+author: michal.kotyla
+layout: post
+published: true
+date: 2021-12-13
+archives: "2021"
+
+tags:
```

Comment:
@artur-rs we should have clear rules about tags
* when we can introduce new tags?
* imx6 i.mx6 - this can be written in multiple way - question is which form works best and which was used before